### PR TITLE
Install to LIBRARY_PREFIX on Windows to match behavior of proj.4-feed…

### PR DESCRIPTION
**Pull request by jedfrechette**
Closes conda-forge/proj.4-feedstock#133

Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by jedfrechette** [comment]
@conda-forge-admin, please rerender

**Comment by xylar** [comment]
@jedfrechette, this package is `noarch: generic`:
https://github.com/conda-forge/proj-data-feedstock/blob/main/recipe/meta.yaml#L15
That means, although it can be installed on Windows, it doesn't get built or tested on Windows.  So the changes you added here will not be used and aren't needed.

Even so, I appreciate you being proactive in trying to improve the package!

**Comment by xylar** [comment]
If this package is not working on Windows, we need to modify it to no longer be `noarch: generic` and build separately for each architecture.  There may be ways we could build one package for all unix and one for Windows, but I'm not particularly adept at that.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -11,7 +11,7 @@ source:
 
 
 build:
-  number: 0
+  number: 1
   noarch: generic
```
Comment (xylar) [inline recipe/meta.yaml:15]: I suggest removing this line:
```suggestion
```
and then rerendering again.

**Comment by jedfrechette** [comment]
@conda-forge-admin, please rerender

**Comment by jedfrechette** [comment]
I've applied the suggested change to remove `noarch: generic`. I'll admit to not being entirely familiar with how these cross-platform builds work, but that makes sense.

As described in the linked issue the proj-data package is broken on Windows. In my local testing on Windows these changes did fix that issue and removed the build test failures I was getting with the existing recipe even before removing `noarch:generic`. However, I guess fixing that build test was just a result of adding a selector to prevent the unix test from trying to run and failing but not actually running the win test? It looks like all the tests are passing correctly now though.

**Comment by xylar** [comment]
@ocefpaf, do you have any tricks up your sleeve that would save us from having to build this package for every unix variant?

**Comment by xylar** [comment]
@conda-forge-admin, please rerender

**Comment by xylar** [comment]
@conda-forge-admin, please rerender

**Comment by xylar** [comment]
Okay, I've followed the approach used in https://github.com/conda-forge/cuda-feedstock and that seems to be working.

**Review by xylar — APPROVED** [review]

**Comment by xylar** [comment]
Thanks for getting this process started @jedfrechette!  I really appreciate it.

**Comment by Nodd** [comment]
Thank you for the fix, it's greatly appreciated! :)

**Comment by xylar** [comment]
@Nodd, keep us posted on whether this fix works for you (once the packages are available).

**Comment by ocefpaf** [comment]
Sorry, I git here too late. This solution is fine but I wonder if the data is in the Windows package, we should maybe change the value of that env var on Windows to point to it instead of change the data package.

**Comment by xylar** [comment]
@ocefpaf, the package size certainly suggests that the data is there.  I'm downloading it now but on a slow train wifi connection.

I agree that pointing the Windows proj package to the same shared directory as unix seems like the easier fix unless that causes some other problem.  We could try to fix that in the proj feedstock and switch this one back to the way it was once we've done that, but there's a risk that Windows ends up broken again if someone has the wrong combination of proj and proj-data.

**Comment by ocefpaf** [comment]
If this is working, leave it. It is not worth rocking the boat again. We are doing OS-noarch anyway, something that we did not have at the time we first made this package and already reduces the build matrix by a lot.

**Comment by Nodd** [comment]
> @Nodd, keep us posted on whether this fix works for you (once the packages are available).

I can confirm that updating the package fixed my problem. Thanks again !

**Pull request merged**